docs/product_search/parameters.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: default
+title: Product Search Parameters
+parent: Product Search
+---
+
+# Product Search Parameters
+Product search is based on query parameters that determine which products are returned and in which order.
+
+### active_promotion
+Accepts: `true` or `false`.<br>
+Passing `true` will return items with a currently active [promotion]({% link docs/reference/objects/product/promotion.md %}#promotionactive).
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search active_promotion: true %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as a query paramameter
+```
+https://mysite.com/search?search[active_promotion]=true
+```
+
+### category
+Accepts any of the Easol predefined categories.<br>
+Will accept either a single category or an array.<br>
+Will return products which match the [category]({% link docs/reference/objects/product/index.md %}#productcategory) passed.
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search category: 'active' %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as a query paramameter
+```
+https://mysite.com/search?search[category]=active
+```
+
+### country
+Accepts any of the Easol predefined [countries]({% link docs/reference/objects/product/index.md %}#productcountry) as either Country Codes or Country Names<br>
+Will accept either a single country or an array.<br>
+Will return products which match the country passed.
+
+##### as a search tag attribute
+{% raw %}
+```
+{% product_search country: 'FR' %}
+{% endproduct_search %}
+
+{% product_search country: ['FR','DE'] %}
+{% endproduct_search %}
+```
+{% endraw %}
+
+##### as a query paramameter
+```
+https://mysite.com/search?search[country]=DE
```

Comment:
according to Theo here:
https://easolhq.slack.com/archives/C03HMNR7WN7/p1667491689028819
query params should be of the form (for array at least)
?search[country][]=Argentina&search[country][]=France

Reply:
Will leave this out for now and update once fixed by the platform team